apps/content/api/portal/issue_reports.py

```
@@ -24,43 +28,89 @@ class IssueReportOutReporter(Schema):
+        asset_id=data.asset_id,
         description=data.description,
     )
+    logger.info(f"Issue report created [report_id={report.id}, user_id={request.user.id}]")
     return 201, report
 
 
-@router.get("issue-reports/", response=list[IssueReportOut])
+@router.get("issue-reports/", response=list[IssueReportOut], auth=ninja_jwt_auth)
 @paginate
 @ordering(ordering_fields=["created_at", "status"])
 def list_issue_reports(request: Request, filters: IssueReportFilter = Query()):
```

Ruff (0.15.12)

[warning] Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)